app/src/main/java/bg/zahov/app/ui/settings/SettingsScreen.kt

```diff
@@ -0,0 +1,282 @@
+                text = theme,
+                dialog = { onDismiss -> changeTheme(onDismiss) })
+        }
+        ColumnTemplate {
+            SettingsText(stringResource(R.string.rest_timer_text))
+            SettingsRadioButton(
+                title = stringResource(R.string.timer_increment_value_text),
+                text = resetTimer.toString(),
+                dialog = { onDismiss -> changeResetTimer(onDismiss) })
+            SettingsSwitchButton(
+                title = stringResource(R.string.vibrate_upon_finish_text),
+                checked = enableVibrate
+            ) { enableVibrateChange() }
+            SettingsRadioButton(
+                title = stringResource(R.string.sound_text),
+                text = sound,
+                dialog = { onDismiss -> changeSound(onDismiss) })
+        }
+        ColumnTemplate {
+            SettingsText(stringResource(R.string.advanced_settings_text))
+            SettingsSwitchButton(
+                title = stringResource(R.string.sync_option_text),
+                text = stringResource(R.string.sync_option_explain),
+                checked = enableSync
+            ) { enableSyncChange() }
+            SettingsSwitchButton(
+                title = stringResource(R.string.show_update_template),
+                text = stringResource(R.string.show_update_template_explain),
+                checked = showUpdate
+            ) { showUpdateChange() }
+            SettingsSwitchButton(
+                title = stringResource(R.string.use_watch_option),
+                checked = enableWatch
+            ) { useWatchChange() }
+        }
+        ColumnTemplate {
+            SettingsButton(stringResource(R.string.github_text)) { github() }
+            SettingsButton(stringResource(R.string.bug_report_text)) { bugReport() }
+        }
+        Column(modifier = Modifier.fillMaxWidth()) {
+            Button(
+                modifier = Modifier
+                    .width(240.dp)
+                    .align(Alignment.CenterHorizontally),
+                onClick = { logout() },
+                colors = ButtonColors(
+                    Color.Red, Color.White, Color.Red, Color.White
+                )
+            ) {
+                Text(text = stringResource(R.string.sign_out_button_text))
+            }
+            Button(
+                modifier = Modifier
+                    .width(240.dp)
+                    .padding(bottom = 80.dp)
+                    .align(Alignment.CenterHorizontally),
+                onClick = { deleteAccount() },
+                colors = ButtonColors(
+                    Color.Red, Color.White, Color.Red, Color.White
+                )
+            ) {
+                Text(text = stringResource(R.string.delete_account_text))
+            }
+        }
+    }
+}
+
+private fun openLink(context: Context, link: String) {
+    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(link))
```

Comment:
`Uri.parse` throws an exception that you should handle here.

Reply:
added a check with toasts to inform the user